portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,29 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  color:lightslategray;
+}
+
+body {
+  background-color:seashell;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+button {
+  font-family: 'Courier New', Courier, monospace;
+  background-color: lightblue;
+  color:lightslategray;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+img {
+  width: 25%;
+  height: 25%;
```

Comment:
Does 'height: 25%' actually change anything here?

Reply:
I just realized it actually doesn't do anything - just got rid of that line!